src/runtime/Value.h

```diff
@@ -431,10 +431,34 @@ inline size_t valueFunctionCopyCount(Value::Type type)
     return s;
 }
 
+inline bool hasCPUWordAlingedSize(Value::Type type)
+{
+#if defined(WALRUS_32)
+    ASSERT(valueStackAllocatedSize(type) == valueSize(type));
+#endif
+    return valueStackAllocatedSize(type) == valueSize(type);
+}
+
+inline bool needsCPUWordAlingedAddress(Value::Type type)
+{
+#if defined(WALRUS_32)
+    // everything is already aligned!
+    return false;
+#else
+    switch (type) {
+    case Value::FuncRef:
+    case Value::ExternRef:
+        return true;
```

Comment:
little bit confused, do these FuncRef and ExternRef need alignment?

Reply:
Arm devices cannot support unaligned pointer well :(
https://embeddedartistry.com/blog/2016/12/26/arm-pointer-alignment-requirements/
+ if we want to use bdwgc later, we need to align pointer in stack

Reply:
Then what about 32bit platform?
This patch marks alignment only for addresses in 64bit platform, but I think that this alignment needs to be applied for 32bit addresses too

Reply:
IMO we don't consider alignment of 32-bit platform
Every value contents have multiple of 4-byte size and
and we have experience that alloca function returns aligned address

Reply:
Oh, I get it :)
Packing approach has this issue only for 64bit platform